Tasks/LambdaDeployFunction/TaskOperations.ts

```diff
@@ -67,6 +67,10 @@ export class TaskOperations {
 
             const response = await this.lambdaClient.updateFunctionCode(updateCodeRequest).promise()
 
+            if (!response.FunctionArn) {
```

Comment:
An empty/undefined `response.FunctionArn` value doesn't seem expected. Is there a reason we wouldn't throw in this case instead?

Reply:
because we didn't throw before, this is supposed to be adding strict mode with as few changes as possible. Will add.